qa/test_template.sh

```diff
@@ -48,24 +48,22 @@ do
         echo "Test variant run: $variant"
         # install packages
         inst=$($topdir/qa/setup_packages.py -i $i -u $pip_packages --cuda ${CUDA_VERSION})
-        if [ -n "$inst" ]
-        then
-        pip install $inst -f /pip-packages
+        if [ -n "$inst" ]; then
+            pip install $inst -f /pip-packages
 
-        # If we just installed tensorflow, we need to reinstall DALI TF plugin
-        if [[ "$inst" == *tensorflow-gpu* ]]; then
-            pip uninstall -y nvidia-dali-tf-plugin || true
-            pip install /opt/dali/nvidia-dali-tf-plugin*.tar.gz
-        fi
+            # If we just installed tensorflow, we need to reinstall DALI TF plugin
+            if [[ "$inst" == *tensorflow* ]]; then
```

Comment:
...wouldn't that react to _anything_ with "tensorflow" in the name? Like some plugins?

Reply:
Yes it would, but we don't get that from $inst (that comes from setup_packages.py).

Reply:
OK then.